.htaccess

```diff
@@ -133,12 +146,12 @@ RedirectMatch "/igbo(\/?)$"            "/keyboards/sil_nigeria_dot"
 RedirectMatch "/khmer(\/?)$"           "/keyboards/khmer_angkor"
 RedirectMatch "/lao(\/?)$"             "/keyboards/basic_kbdlao"
 RedirectMatch "/malayalam(\/?)$"       "/keyboards/basic_kbdinmal"
-RedirectMatch "/maltese(\/?)$"         "/keyboards/maltese"
+RedirectMatch "^/maltese(\/?)$"         "/keyboards/maltese"
 RedirectMatch "/marathi(\/?)$"         "/keyboards/basic_kbdinmar"
 RedirectMatch "/mongolian(\/?)$"       "/keyboards/basic_kbdmon"
 RedirectMatch "/nepali(\/?)$"          "/keyboards/basic_kbdnepr"
 RedirectMatch "/oriya(\/?)$"           "/keyboards/basic_kbdinori"
-RedirectMatch "/rawang(\/?)$"          "/keyboards/rawang"
+RedirectMatch "^/rawang(\/?)$"          "/keyboards/rawang"
```

Comment:
This differs from all the others here. Afaict they should all start with ^ and not /

Reply:
ok, standardized all the keyboard redirects to use `^/foo(/?)$`